src/main/java/github/grace000/erversay/ConnectionWrapper.java

```diff
@@ -0,0 +1,6 @@
+package github.grace000.erversay;
+
+public interface ConnectionWrapper {
```

Comment:
Is this used outside of your tests?

Reply:
I am implementing the interface outside of my tests.